### DISCUSSION TITLE
Standalone C++ package for moonshine

### DISCUSSION THREAD

**Discussion by royshil** [moonshine-ai/moonshine#53] [Show and tell]
Hello 
I've created a simple C++ inference code for moonshine https://github.com/locaal-ai/moonshine.cpp
It only depends on ONNXRuntime (which is fetched in build time by CMake).
Makes it easy to integrate into C++ projects with very minimal dependency, using CMake.
Enjoy!

**Comment by briangavin** [comment]
I love this idea.   It is much easier to package applications using c++.   It looks you have not added VAD detection yet.   This would help improve the quality by reducing hallucinations.
